core/serialization.c

```diff
@@ -115,15 +135,21 @@ Shape *serialization_load_shape(Stream *s,
     stream_free(s);
 
     // shrink box once all blocks were added to update box origin
```

Comment:
Please remove this comment if the associated code was removed (or move it where the new calls are done)